backend/src/main/java/feedzupzup/backend/feedback/domain/Feedback.java

```diff
@@ -71,4 +82,16 @@ public void updateLikeCount(int likeCount) {
         }
         this.likeCount = likeCount;
     }
+
+    public boolean isConfirmed() {
+        return this.status == ProcessStatus.CONFIRMED;
+    }
+
+    public boolean isWaiting() {
+        return this.status == ProcessStatus.WAITING;
```

Comment:
p2)
```suggestion
    public boolean isEqualToStatus(ProcessStatus processStatus) {
        return this.status == processStatus;
    }
```
새로운 상태가 추가될걸 가정하면, 위 코드가 더 범용적으로 쓰일 수 있을 것 같아요